hazelcast/src/main/java/com/hazelcast/monitor/NearCacheStats.java

```diff
@@ -59,4 +59,11 @@
      * @return hit/miss ratio of the locally owned entries.
      */
     double getRatio();
+
+    /**
+     * Returns the number of evictions of locally owned entries.
```

Comment:
I used the same phrase like in all other methods. I think it makes sense since the statistics all refer to the entries in the local Near Cache, not to the backing data structure (which can be of course remote or partly local).

But you are right about eviction vs. expiration. At the moment there is no difference between that in this PR (eviction, TTL and max-idle should all increase that counter).

Would you be fine with the enhancement if I would split that up?